howto/stream/oob-data.md

```diff
@@ -106,15 +106,169 @@ A trivial example to simulate the data transmission between an Anbox container a
 
- - The socket connection
- - Polling events
- - Forwarding data between both ends
+ - Accepting connection requests from an Android application
+ - Connecting to one specific data channel via the Unix domain socket exposed by the Anbox WebRTC platform
+ - Passing the connected socket as a file descriptor to the Android application
 
-A system service named `anbox-data-proxy`, which covers all the above functionalities, will be implemented in the 1.16.0 release of Anbox Cloud. It makes use of [`gbinder`](https://github.com/mer-hybris/libgbinder) to set up a communication bridge and provide reliable and high-efficiency data transmission between the Android system and the Anbox WebRTC platform.
+The `anbox-webrtc-data-proxy` system daemon runs in the Anbox container and registers an Android system service named `org.anbox.webrtc.IDataProxyService`. This service allows Android developers to easily make use of binder interprocess communication (IPC) for data communication between an Android application and the Anbox WebRTC platform through a file descriptor.
+
+[note type="information" status="Note"]To interact with the `org.anbox.webrtc.IDataProxyService` system service, the Android application must be [installed as a system app](https://discourse.ubuntu.com/t/how-to-install-an-apk-as-a-system-app/27086). [/note]
+
+#### Get notified about the availability of data channels
+
+To get notified about the availability of data channels, an Android application can register the following broadcast receiver in the `AndroidManifest.xml` file :
+
+    ```
+        <receiver
+            android:name=".DataChannelEventReceiver"
+            android:enabled="true"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="com.canonical.anbox.BROADCAST_DATA_CHANNELS_STATUS"/>
+            </intent-filter>
+        </receiver>
+
+    ```
+
+Then a broadcast which carries the following parameters is sent out to the Android application on the change in the availability of data channels.
+
+| Parameters               | Type                                            | Description                                                                                                                                                                |
+| ------------------------ | ----------------------------------------------- | -----------------------------------------------                                                                                                                            |
+| `event`                  | String                                          | Can be `"created"` or `"destroyed"` respectively denotes the data channels are created and open for Android applications to use, or the data channels are closed and destroyed. |
```

Comment:
```suggestion
| `event`                  | string                                          | Can be `created` (which means the data channels are created and open for Android applications to use) or `destroyed` (which means that the data channels are closed and destroyed) |
```

Reply:
I assume that the strings don't contain the double quotes? So it's, for example, `created` and not `"created"`?